src/ios/CameraSessionManager.m

```diff
@@ -109,6 +116,20 @@ - (void) setupSession:(NSString *)defaultCamera completion:(void(^)(BOOL started
     }];
 }
 
+- (AVCaptureSessionPreset) calculateResolution:(NSInteger)height {
```

Comment:
width in android, height in ios, can we have more information about this inversion

Reply:
@dinitri-ragoo height should be maxSize here